package.json

```diff
@@ -53,21 +53,22 @@
     "@commitlint/cli": "^8.1.0",
     "@commitlint/config-conventional": "^8.1.0",
     "@commitlint/config-lerna-scopes": "^8.1.0",
-    "@typescript-eslint/parser": "^2.16.0",
     "@typescript-eslint/eslint-plugin": "^2.16.0",
+    "@typescript-eslint/parser": "^2.16.0",
```

Comment:
Why change in this file? can you remove this file from the commit

Reply:
done